Makefile

```diff
@@ -74,7 +74,7 @@ generators:
 	python -m autonomy.cli hash all
 	python scripts/generate_api_documentation.py
 	python scripts/check_copyright.py
-	python scripts/check_doc_ipfs_hashes.py
+	python scripts/fix_doc_ipfs_hashes.py
```

Comment:
Should be updated in tox and ci too?

Reply:
It depends: do we want this as a CI check? At the moment this is tested as a normal unit test inside tests/test_docs, and the dev needs to run make generators to fix the hashes.